canarytokens/aws_infra/operations.py

```diff
@@ -42,6 +42,17 @@
 AWS_INFRA_AWS_ACCOUNT = settings.AWS_INFRA_AWS_ACCOUNT
 HANDLE_RESPONSE_TIMEOUT = 300  # seconds
 
+# map to user-friendly error messages
+service_error_map = {
```

Comment:
I don't think we need the comment here, and since it's a constant let's UPPERCASE it
```suggestion
SERVICE_ERROR_MESSAGE_MAP = {
```